### ISSUE TITLE
This isn't working properly in my xcode :( 

### ISSUE THREAD

**Issue by podiyaaa** [mcichecki/Combine-MVVM#2]
I try this example with my xcode and it doesn't work for me. will you be able to help me ?

**Comment by shinrenpan** [comment]
All AnyCancellable should stored in an Set<AnyCancellable>

try
```swift
class LoginViewController: UIViewController {
    ...
    private var bindings = Set<AnyCancellable>()
}

contentView.loginTextField.textPublisher
  .receive(on: DispatchQueue.main)
  .assign(to: \.login, on: viewModel)
  .store(in: &bindings)
```

**Comment by mcichecki** [comment]
@shinrenpan thanks, fix (#3) is already available on master branch.
